Cargo.toml

```diff
@@ -226,9 +225,7 @@ uuid = { version = "1.1", features = ["v4"] }
 qwal = { git = "https://github.com/tremor-rs/qwal" }
 
 [dev-dependencies]
-serial_test = { version = "=0.8", features = [
-  "logging",
-] }
+serial_test = { version = "=0.8", features = ["logging"] }
```

Comment:
is this supposed to be in this PR? seem unrelatedd

Reply:
ja it sucks when that happens -.- we've not found a way yet to unify this the same way it works with `cargo fmt` :(